Jenkinsfile

```diff
@@ -128,45 +129,36 @@ pipeline {
             }
         }
 
-/////////////////////////////////////smoke test//////////////////////////////////////////////////////////
-
-
 
-stage('Smoke Test') {
-         steps {
+	stage('Smoke Test') {
+          steps {
+		  script {
 
-	  script {
+		        serviceName = 'maingateway-service'
+			smokeTestOperation='cicd/maingateway/profile/11111?alertType=ACCIDENT'
+			makeGetRequest("http://${serviceName}/${smokeTestOperation}")
 
-	        serviceName = 'maingateway-service'
-		smokeTestOperation='cicd/maingateway/profile/11111?alertType=ACCIDENT'
-		makeGetRequest("http://${serviceName}/${smokeTestOperation}")
 
-
-		serviceName = 'fisuser-service'
-		smokeTestOperation='cicd/user/profile/11111'
-		makeGetRequest("http://${serviceName}/${smokeTestOperation}")
+			serviceName = 'fisuser-service'
+			smokeTestOperation='cicd/user/profile/11111'
+			makeGetRequest("http://${serviceName}/${smokeTestOperation}")
                 
 
-		serviceName = 'fisalert-service'
-		smokeTestOperation='cicd/alert'
-		body = ''' { "alertType": "ACCIDENT",  "firstName": "Abdul Hameed",  "date": "11/8/2019",  "phone": "78135955",  "email": "ahameed@redhat.com",  "description": "test"} '''
-		makePostRequest("http://${serviceName}/${smokeTestOperation}", body,'POST')
+			serviceName = 'fisalert-service'
+			smokeTestOperation='cicd/alert'
+			body = ''' { "alertType": "ACCIDENT",  "firstName": "Abdul Hameed",  "date": "11/8/2019",  "phone": "78135955",  "email": "ahameed@redhat.com",  "description": "test"} '''
+			makePostRequest("http://${serviceName}/${smokeTestOperation}", body,'POST')
 
-		serviceName = 'nodejsalert-ui'
-	        makeGetRequest("http://${serviceName}:8080")
+			serviceName = 'nodejsalert-ui'
+		        makeGetRequest("http://${serviceName}:8080")
 
-	  }
+		  }
 
  		
 	}
 		
    	}
```

Comment:
I suggest to make these format/indentation to align with the overall coding style.

Reply:
This is already done by you in earlier PR, so this will resolved once code merged.